src/NodaTime-Core.sln

```diff
@@ -15,208 +15,44 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "NodaTime.Serialization.Json
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "NodaTime.Serialization.Test", "NodaTime.Serialization.Test\NodaTime.Serialization.Test.csproj", "{DA5AF694-C7DF-4989-947D-355C4C5F912B}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "NodaTime.Serialization.ModelBinding", "NodaTime.Serialization.ModelBinding\NodaTime.Serialization.ModelBinding.csproj", "{292A34C7-3E56-48BE-B067-BE2EE2C54D4F}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
-		Debug Portable|Any CPU = Debug Portable|Any CPU
```

Comment:
Not sure why so many projects being removed from the sln file when this PR is supposed to be adding a new project.